temporalcli/commandsgen/commands.yml

```diff
@@ -143,11 +145,44 @@ commands:
         description: Active environment name (`ENV`).
         default: default
         env: TEMPORAL_ENV
+        # TODO(cretz): Deprecate when `config` GA
+        # deprecated: |
+        #   Use `profile` instead. If an env file is present, it will take
+        #   precedent over `config` file or config environment variables.
       - name: env-file
         type: string
         description: |
           Path to environment settings file.
           Defaults to `$HOME/.config/temporalio/temporal.yaml`.
+        # TODO(cretz): Deprecate when `config` GA
+        # deprecated: |
+        #   Use `config-file` instead. If an env file is present, it will take
+        #   precedent over `config` file or config environment variables.
+      - name: config-file
+        type: string
+        description: |
+          File path to read TOML config from, defaults to
+          `$CONFIG_PATH/temporal/temporal.toml` where `$CONFIG_PATH` is defined
+          as `$HOME/.config` on Unix, "$HOME/Library/Application Support" on
+          macOS, and %AppData% on Windows.
+        experimental: true
+        implied-env: TEMPORAL_CONFIG_FILE
+      - name: profile
+        type: string
+        description: Profile to use for config file.
```

Comment:
Should this use `default` as the default?

Reply:
No because the default is actually determined by the Go SDK not by the CLI. A CLI flag (whether set via "default" or explicitly) would override where it may be otherwise set, e.g. env var or the actual default.